src/sql/base/browser/ui/taskbar/overflowActionbarStyles.ts

```diff
@@ -13,28 +13,28 @@ import { focusBorder } from 'vs/platform/theme/common/colorRegistry';
 registerThemingParticipant((theme: IColorTheme, collector: ICssStyleCollector) => {
 	const overflowBackground = theme.getColor(EDITOR_PANE_BACKGROUND);
 	if (overflowBackground) {
-		collector.addRule(`.carbon-taskbar .overflow {
+		collector.addRule(`.toolbarOverflow {
```

Comment:
Why did you remove the carbon-taskbar scoping?

Reply:
I see - it appears because you want these rules to apply to both the ModelView component and base component? So in general we should really be trying to avoid that - having styles defined here that are used in other completely separate places in the app make it difficult to track dependencies and so it's easy to envision this being changed and breaking the ModelView toolbar but not noticing immediately because it's not clear that this is tied directly to the ModelView toolbar.

If you take my suggestion of re-using the base component in the ModelView component that's less of an issue since you can keep this scoped down and know that the style will apply anywhere that uses this component.

Reply:
Added back the carbon-taskbar scoping and now have the ModelView toolbar theming set separately